docs/source/Plugin/_Plugin.rst

```diff
@@ -207,30 +207,25 @@ See :ref:`Rules: Formatting refered values <Formatting values>` on how this can
 Just remember such formatting cannot "make up" more decimals than what was set here in the task setup.
 
 
-
-
-
-
-
-
-
 List of official plugins
 ========================
 
 There are different released versions of ESP Easy:
 
-:green:`NORMAL` is the stable release, you can consider these plugins reliable and you can use these in production.
+:green:`NORMAL` is the regular set of plugins, you can consider these plugins stable with all secondary features enabled, like I2C multiplexer, RTTL, DEBUG logging, etc.
 
-:yellow:`TESTING` (split into A/B/C/D/E sets) with new plugins that have not yet been fully tested and proven stable. Because of space limitations, this collection is split into 5 sets. When only TESTING is mentioned, the plugin is available in all TESTING builds.
+:yellow:`COLLECTION` (split into A/B/C/D/E sets) with plugins that don't fit into the NORMAL builds. Because of space limitations, this collection is split into 5 sets. When only COLLECTION is mentioned, the plugin is available in **all** COLLECTION builds. Also, some features are disabled to save space in the .bin files, like the I2C multiplexer feature, RTTTL, tooltips, and some DEBUG logging.
```

Comment:
Change "5" into "a number of".
Such specific numbers will get out of sync when we need "Collection F" etc.

Reply:
Ok, but then it could also be updated to 6 😉

Reply:
I should, but will be forgotten...

Reply:
It's on my TODO list to update once we add COLLECTION F 😃